packages/material-ui/.prettierignore

```diff
@@ -0,0 +1 @@
+*.test.tsx
```

Comment:
We're not using prettier, so we shouldn't need a .prettierignore

Reply:
I use it. There is no way to ignore some files from VSCode prettier extension settings. We can place that file on top level